### PR TITLE
New pull request

### PR DESCRIPTION
**Pull request by VanHoutte1337**
I use a tool to format my code, i hope this doesn't bother you.

### CODE REVIEW

File: src/antwerpen/coordinate/coordinate.service.ts [hunk 1]
```diff
@@ -105,7 +105,7 @@ export class CoordinateService {
                 return Promise.resolve(undefined);
             }
 
-            const doc = response.results[0];
+            const doc = response.features[0];
```
Comment (jsebrech) [inline src/antwerpen/coordinate/coordinate.service.ts]: Why is this change needed? The service still seems to return "results"?

Comment (VanHoutte1337) [reply src/antwerpen/coordinate/coordinate.service.ts]: If i run the following:
curl -X GET \
  'https://querybylocation.antwerpen.be/querybylocation/pointnearby?url=https://geoint.antwerpen.be/arcgissql/rest/services/P_Stad/Mobiliteit/MapServer/6/query&range=100&layerids=6&sr=4326&x=51.204689371609454&y=4.441109419130953' \
  -H 'Postman-Token: 6e182229-f838-4c9a-9fd6-6ade04d49a6b' \
  -H 'cache-control: no-cache'

I receive the following result:
`{
    "displayFieldName": "GISID",
    "fieldAliases": {
        "OBJECTID": "OBJECTID",
        "GISID": "GISID",
        "STRAAT": "STRAAT",
        "POSTCODE": "POSTCODE",
        "EBDD": "EBDD",
        "STATUS": "STATUS",
        "SHAPE.STLength()": "SHAPE.STLength()"
    },
    "geometryType": "esriGeometryPolyline",
    "spatialReference": {
        "wkid": 4326,
        "latestWkid": 4326
    },
    "fields": [
        {
            "name": "OBJECTID",
            "type": "esriFieldTypeOID",
            "alias": "OBJECTID"
        },
        {
            "name": "GISID",
            "type": "esriFieldTypeString",
            "alias": "GISID",
            "length": 50
        },
        {
            "name": "STRAAT",
            "type": "esriFieldTypeString",
            "alias": "STRAAT",
            "length": 80
        },
        {
            "name": "POSTCODE",
            "type": "esriFieldTypeString",
            "alias": "POSTCODE",
            "length": 4
        },
        {
            "name": "EBDD",
            "type": "esriFieldTypeDate",
            "alias": "EBDD",
            "length": 8
        },
        {
            "name": "STATUS",
            "type": "esriFieldTypeString",
            "alias": "STATUS",
            "length": 50
        },
        {
            "name": "SHAPE.STLength()",
            "type": "esriFieldTypeDouble",
            "alias": "SHAPE.STLength()"
        }
    ],
    "features": []
}`

features contains the returned gis objects

Comment (jsebrech) [reply src/antwerpen/coordinate/coordinate.service.ts]: So, this is a bug where before it could not find bicycle routes and now it can? Next time please add a description of the bug in the PR or create a corresponding issue.

Comment (VanHoutte1337) [reply src/antwerpen/coordinate/coordinate.service.ts]: Well I'm not sure this was a bug, is it possible they changed the layout for a call? I found this by accident.

**Comment by jsebrech** [comment]
Published as v2.2.1
